src/presentation/http/router/noteList.test.ts

```diff
@@ -0,0 +1,116 @@
+import userSessions from '@tests/test-data/userSessions.json';
+import { describe, test, expect, beforeAll } from 'vitest';
+
+
+/**
+ * Access token that will be used for Auhorization header
+ */
+let accessToken = '';
+
+
+describe('NoteList API', () => {
+  beforeAll(async () => {
+    /**
+     * userId for authorization
+     */
+    const userId = userSessions[0]['user_id'];
+
+    accessToken = global.auth(userId);
+  });
+
+  describe('GET /notes?page', () => {
+    test('Returns noteList with specified length (not for last page)', async () => {
+      const expectedStatus = 200;
+      const portionSize = 30;
```

Comment:
you sure you need this much data to test this case?

Reply:
page contains 30 notes by default so i think we need at least more than 30?